### PR TITLE
fix(CI): add CI workflow and remove old tests workflow

### PR DESCRIPTION
**Pull request by jbeveland27**
Noticed our tests workflow fails, but seems like it's due to using an old file that relied on Node 10. This just pulls in the CI workflow from the template project, which feels more complete.

### CODE REVIEW

**Comment by jbeveland27** [comment]
lint + test issues resolved. CI workflow is now passing 🎉

File: apolloschurchapp/src/theme/theme.js [hunk 1]
```diff
@@ -208,33 +236,7 @@ export const overrides = {
...
-                  We hope you feel encouraged by God’s presence in your life and
-                  his love for you today!
-                </BodyText>
-              </Text>
-            }
-          />
-        )}
-      />
-    ),
+    AddPrayerComponent,
```
Comment (jbeveland27) [inline apolloschurchapp/src/theme/theme.js:239]: This is the main change I'm not super certain about. If someone can just sanity check that this makes sense that'd be good.

Comment (conrad-vanl) [reply apolloschurchapp/src/theme/theme.js:239]: Yeah, this looks fine.

**Comment by conrad-vanl** [comment]
@vinnyjth if you could double check things on the API-side won't interfere with our upgrade process - mainly making sure CI process and such is in line with core. Then I think we should be good here.

**Comment by jbeveland27** [comment]
@conrad-vanl It feels like we were already missing some things from the upgrade process. Best I could tell, we're on v2.22.0 currently, and some of the failing tests/lint issues were not present in the core v2.22.0 tag, but were present in our repo. This PR is mostly an attempt to bring some alignment with those things and have a working CI workflow for PRs.

**Pull request closed without merging**